deliverables/movie-list/movie-list.md

```diff
@@ -1,12 +1,12 @@
 # movie list application
```

Comment:
All the file in `delivrables` and `lesson-plans` was already there and I never took the time to check them, feel free to keep them or not based on your approach for the module.